testinfra/modules/interface.py

```diff
@@ -49,30 +59,74 @@ def get_module_class(cls, host):
             return BSDInterface
         raise NotImplementedError
 
+    @classmethod
+    def names(cls):
+        """Return the names of all the interfaces.
+
+        >>> host.interface.names()
+        ['lo', 'tunl0', 'ip6tnl0', 'eth0']
+        """
+        raise NotImplementedError
+
+    @classmethod
+    def default(cls, family=None):
+        """Return the default interface.
+
+        >>> host.interface.default()
+        <interface eth0>
+
+        Optionally, the protocol family to use can be enforced.
+
+        >>> host.interface.default("inet6")
+        None
+        """
+        raise NotImplementedError
+
 
 class LinuxInterface(Interface):
     @cached_property
-    def _ip(self):
-        return self.find_command("ip")
+    def _ip(self, include_family=True):
+        ip_cmd = self.find_command("ip")
+        if include_family and self.family is not None:
+            ip_cmd = f"{ip_cmd} -f {self.family}"
```

Comment:
The ``include_family`` parameter is always True and will always be True since you can't call a ``property`` with parameters. So you can safely drop this variable ;)

Reply:
Indeed! 😓